conda.recipe/meta.yaml

```diff
@@ -67,6 +67,7 @@ test:
     - selenium >=3
     - pytest-xdist
     - pytest-rerunfailures
+    - requests >=1.2.3
```

Comment:
This one is still needed I think, for tests. It's the runtime dependency in `setup.py` I wanted to get rid of. Tho if the tests pass I guess I am wrong and this can go!